Code/server/GameServer.cpp

```diff
@@ -383,6 +391,129 @@ void GameServer::BindServerCommands()
                 out->error("Day must be between 0 and 31, month must be between 0 and 11, and year must be between 0 and 999.");
             }
         });
+
+    m_commands.RegisterCommand<std::string>(
+        "AddAdmin", "Add admin privileges from player", [&](Console::ArgStack& aStack) {
+            auto out = spdlog::get("ConOut");
+
+            const auto& cUsername = aStack.Pop<String>();
+
+            bool playerFound = false;
+
+            PlayerManager::Get()->ForEach([&](const Player* apPlayer) {
+                if (apPlayer->GetUsername() == cUsername)
+                {
+                    AddAdminSession(apPlayer->GetConnectionId());
+                    out->info("{} admin privileges added", cUsername.c_str());
+                    playerFound = true;
+                    return;
+                }
+            });
+
+            if (!playerFound)
+            {
+                // space in username handling
+                String backupUsername = cUsername;
```

Comment:
Should factor out the user name sanitizing / cleaning into a separate utility fn.